src/Badge/Badge.tsx

```diff
@@ -44,11 +48,12 @@ export const Badge: RneFunctionComponent<BadgeProps> = ({
   textProps,
   badgeStyle,
   onPress,
-  Component = onPress ? TouchableOpacity : View,
+  Component = onPress ? Pressable : View,
   value,
   theme,
   status = 'primary',
-  ...attributes
+  pressableProps,
+  ...props
```

Comment:
```suggestion
  ...rest
```

Pls make this change at all places